lib/register_sources_bods/config/adapters.rb

```diff
@@ -12,6 +14,11 @@ module Adapters
       HTTP_ADAPTER    = RegisterCommon::Adapters::HttpAdapter.new
       KINESIS_ADAPTER = RegisterCommon::Adapters::KinesisAdapter.new(credentials: AWS_CREDENTIALS)
       S3_ADAPTER      = RegisterCommon::Adapters::S3Adapter.new(credentials: AWS_CREDENTIALS)
+      REDIS_ADAPTER   = RegisterCommon::Adapters::RedisAdapter.new(
```

Comment:
Could I please confirm whether this will mean Register will also have to start using Redis—even in Heroku? Or will it only affect Transformers and Ingesters?

Reply:
It's only used by the ingester and transformer from register-sources-bods, Register won't use it

Reply:
Thanks.